src/main/java/no/nav/familie/integrasjoner/client/rest/ArbeidOgInntektClient.kt

```diff
@@ -0,0 +1,47 @@
+package no.nav.familie.integrasjoner.client.rest
+
+import no.nav.familie.http.client.AbstractRestClient
+import no.nav.familie.integrasjoner.felles.OppslagException
+import no.nav.familie.kontrakter.felles.PersonIdent
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.http.HttpHeaders
+import org.springframework.http.HttpStatus
+import org.springframework.http.MediaType
+import org.springframework.stereotype.Component
+import org.springframework.web.client.RestOperations
+import org.springframework.web.util.UriComponentsBuilder
+import java.net.URI
+
+@Component
+class ArbeidOgInntektClient(
+    @Value("\${ARBEID_INNTEKT_URL}") private val uri: URI,
+    @Qualifier("jwtBearer") private val restOperations: RestOperations,
+) : AbstractRestClient(restOperations, "ainntekt") {
+    private val redirectUri =
+        UriComponentsBuilder
+            .fromUri(uri)
+            .pathSegment("api/v2/redirect/sok/a-inntekt")
+            .build()
+            .toUri()
+
+    fun hentUrlTilArbeidOgInntekt(personIdent: PersonIdent): String =
+        try {
+            getForEntity(
+                redirectUri,
+                HttpHeaders().apply {
+                    accept = listOf(MediaType.TEXT_PLAIN)
+                    set("Nav-Personident", personIdent.ident)
```

Comment:
Sikker på at de ikke tilbyr en tjeneste hvor man slipper å sette en header? Siden det er stor fare for at headere logges i firewall o.l. ?